bulbs/content/tasks.py

```diff
@@ -199,7 +201,9 @@ def post_to_instant_articles_api(content_pk):
                 source,
                 fb_page_id,
                 fb_api_url,
-                fb_token_path)
+                fb_token_path,
+                fb_dev_mode,
```

Comment:
Agreed. But also confusing/error-prone to have to set `FACEBOOK_API_ENV = "production"` in order to test IA submissions even in a testing environment.
What about ```if environment == "production" or fb_dev_mode```?